.infrastructure/helm-chart/todoapp/templates/rbac.yml

```diff
@@ -0,0 +1,30 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{ .Values.todoapp.RBAC.name }}
+  namespace: {{ .Values.todoapp.namespace }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: {{ .Values.todoapp.namespace }}
+  name: {{ .Values.todoapp.RBAC.name }}
```

Comment:
Ensure that the `todoapp.RBAC.name` value is defined in your `values.yaml` file. If it is not set, this could lead to errors during deployment as the Role name would be undefined.